### DISCUSSION TITLE
Duda existencial

### DISCUSSION THREAD

**Discussion by LucasManrique** [Launch-X-Latam/MisionFrontEnd#173] [Q&A]
Hola, como estan?, quería preguntar si todavía falta que suban alguna grabación de las transmisiones. No tuve tiempo de entrar a todas y estoy un poco perdido. Muchas Gracias.

**Comment by sudoagile** [comment]
Hola, creo que estos son los videos, saludos
https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a
